<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Worker</class>
 <widget class="QMainWindow" name="Worker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Worker</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">#workerWidget,#orderWidget{
border-image:url(:/HX/images/维修工.jpg)
}

.QGroupBox{
border:2px solid blue;
background-color:rgb(100,100,200,30%);
border-radius:20%;
}

.QPushButton{
background-color:rgb(100,200,255,50%);
border-radius:10%;
width:70px;
height:30px;
}

.QLineEdit,.QComboBox,.QSpinBox{
background-color:rgb(200,200,200,50%);
height:30px;
margin:0;
}

.QLabel
{
color:blue;
}

QTabWidget::tab-bar {
    left: 5px;
    alignment: center;
}

QTabWidget::pane {
    border: 1px solid #9B9B9B;
    position: absolute;
    top: -1px;
}

QTabBar::tab {
    background: QLinearGradient(x1:0,y1:0,x2:0,y2:1,stop:0 #E1E1E1,stop:0.4 #DDDDDD,stop:0.5 #D8D8D8,stop:1 #D3D3D3);
    border-left: 1px solid #C4C4C3;
    border-right: 1px solid #C4C4C3;
    border-top: 1px solid #C4C4C3;
    border-bottom: 1px solid #9B9B9B;
    border-top-left-radius: 4px;
    border-top-right-radius: 4px;
    min-width: 100px;
    padding: 5px;
}

QTabBar::tab:selected, QTabBar::tab:hover {
    background: QLinearGradient(x1:0,y1:0,x2:0,y2:1,stop:0 #fafafa,stop:0.4 #f4f4f4,stop:0.5 #e7e7e7,stop:1 #fafafa);
}

QTabBar::tab:selected {
    border-left-color: #9B9B9B;
    border-right-color: #9B9B9B;
    border-top-color: #9B9B9B;
    border-bottom-color: #fafafa;
    margin-left: -3px;
    margin-right: -3px;
}

QTabBar::tab:!selected {
    margin-top: 2px;
}

QTabBar::tab:first:selected {
    margin-left: 0;
}

QTabBar::tab:last:selected {
    margin-right: 0;
}

QTabBar::tab:only-one {
    margin: 0;
}</string>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="documentMode">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="workerWidget">
       <attribute name="title">
        <string>个人信息管理</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="2">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>158</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>159</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="1">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string/>
          </property>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="font">
              <font>
               <family>华文行楷</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>工        号</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="WorkerID">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_2">
             <property name="font">
              <font>
               <family>华文行楷</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>姓        名</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="name"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_8">
             <property name="font">
              <font>
               <family>华文行楷</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>身份证号</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="ID">
             <property name="maxLength">
              <number>18</number>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_3">
             <property name="font">
              <font>
               <family>华文行楷</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>工        龄</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QSpinBox" name="Age">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_6">
             <property name="font">
              <font>
               <family>华文行楷</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>工作状态</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QComboBox" name="comboBox">
             <item>
              <property name="text">
               <string>空闲中</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>维修中</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_4">
             <property name="font">
              <font>
               <family>华文行楷</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>业        绩</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QSpinBox" name="timeBox">
             <property name="readOnly">
              <bool>true</bool>
             </property>
             <property name="maximum">
              <number>999999999</number>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="label_5">
             <property name="font">
              <font>
               <family>华文行楷</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>手  机  号</string>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="QLineEdit" name="phone"/>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string/>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="Save">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>保存</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="Cancel">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>取消</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="3" column="1">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="0">
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="2">
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="orderWidget">
       <attribute name="title">
        <string>订单管理</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTableView" name="orderView"/>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string/>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="Finish">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>完成订单</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_5">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="orderCancel">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>取消订单</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="HX.qrc"/>
 </resources>
 <connections/>
</ui>
